lib/get-config.js

```diff
@@ -411,9 +393,9 @@ function getProjectConfig(workingDir, options) {
     ensureRootProperties(config, source);
     validateRootProperties(source);
 
-    config.plugins = processPlugins(workingDir, source.plugins, options);
-    config.loadedRules = processLoadedRules(workingDir, config, options);
-    config.loadedConfigurations = processLoadedConfigurations(workingDir, config, options);
+    config.plugins = await processPlugins(workingDir, source.plugins, options);
+    config.loadedRules = await processLoadedRules(workingDir, config, options);
```

Comment:
Looks like we could do it in parallel (promise all)

Reply:
I just tried this and the tests all pass:

```js
await Promise.all([
      (config.plugins = await processPlugins(workingDir, source.plugins, options)),
      (config.loadedRules = await processLoadedRules(workingDir, config, options)),
      (config.loadedConfigurations = await processLoadedConfigurations(
        workingDir,
        config,
        options
      )),
]);
```

However, I'm still a bit concerned that this might not be safe? Don't we depend on plugins to be loaded first before processing loaded rules? Either way, let's consider this optimization in a separate PR due to the risk.